tests/integration/modules/test_vmc_org_users.py

```diff
@@ -0,0 +1,46 @@
+"""
+    Integration Tests for vmc_org_users execution module
+"""
+import json
+
+import pytest
+
+
+@pytest.fixture
+def vmc_common_data(vmc_connect):
+    data = vmc_connect.copy()
+    data.pop("hostname")
+    data.pop("sddc_id")
+    data.pop("vcenter_hostname")
+    data["hostname"] = vmc_connect["authorization_host"]
+    return data
+
+
+def test_sddc_smoke_test(salt_call_cli, vmc_common_data):
+    # get the org users list
+    ret = salt_call_cli.run("vmc_org_users.list", **vmc_common_data)
+    result_as_json = ret.json
+    assert "error" not in result_as_json
+    existing_org_users = len(result_as_json["results"])
+
+    # Add a new user to the org
+    user_names = ["test@vmware.com"]
+    organization_roles = [
+        {"name": "org_member", "expiresAt": None},
```

Comment:
I get an "illegal request body, illegal value on field: organization roles, illegal value on field: null, illegal value on field: expiresAt" from this

Reply:
looks like by removing `expiresAt` from the role it works.

Actually using this during testing, I can't help but think that the UX here could use some attention.

I'd expect something more like:

```
user.add(
    email="test@vmware.com",
    roles=["org_member", "developer"],
)
```

It's entirely possible that I'm missing something with the way the API functions (I took a look at the API docs but they're pretty sparse)

Reply:
we implemented the functionality based on the VMC APIs

Reply:
corrected the tests.